specification/overview.md

```diff
@@ -193,20 +193,23 @@ OpenTelemetry defines the naming convention for metric names as well as a
 well-known metric names in [Semantic Conventions](data-semantic-conventions.md)
 document.
 
-## DistributedContext
+## CorrelationContext
 
-The **DistributedContext** exists to store labels that describe the context of an operation an application performs. It is intended to enable context that are custom to the application or integrations in contrast to other contexts, such as `SpanContext`. Only one **DistributedContext** should be associated with any particular operation.
+In addition to trace propagation, OpenTelemetry provides a simple mechanism for propagating indexes, called **CorrelationContext**. The **CorrelationContext** is intended for indexing observability events in one service with attributes provided by a prior service in the same transaction. This helps to establish a causal relationship between these events. For example, determining that a particular browser version is associated with a failure in an image processing service.
 
-For example, a web service can benefit from including context around what service has sent the request. Or a SaaS provider can include context about the API user or token that is responsible for that request. These values can be consumed from **DistributedContext** and used as an additional dimension for a metric, or additional context for logs and traces.
+The **CorrelationContext** is based on the [W3C Correlation-Context specification](https://w3c.github.io/correlation-context/), and implements the protocol as it is defined in that working group. While **CorrelationContext** can be used to prototype other cross-cutting concerns, this mechanism is primarily intended to convey values for the OpenTelemetry observability systems.
 
-**DistributedContext** is a collection of key-value `Entry` pairs, with each key of associated with exactly one value. **DistributedContext** is serializable,
-to facilitate propagating it not only inside the process but also across process boundaries.
+For backwards compatibility, OpenTracing Baggage is propagated as **CorrelationContext** when using the OpenTracing bridge. New concerns with different criteria should be modeled separately, using the same underlying context propagation layer as building blocks.
```

Comment:
I think this could use a little more explanation (but it's nice). Maybe something like "Users of OpenTracing baggage should consider creating new a new cross-cutting concern to cover their use-case; they may benefit from the W3C encoding format but use a new HTTP header to convey data throughout a distributed trace."

Reply:
Also I'd move this paragraph down to the bottom of this section, so that "For example" follows the text above.

Reply:
Not sure creating (or advising to create) a new cross-cutting concern is a good idea. I'd like that users of the OT shim get the very same thing as OTel/CorrelationContext and makes things super easy to interoperate. In that regard, I feel using the same W3C encoding is enough.

Reply:
I updated this section to incorporate the added details around implementing a new cross cutting concern separately from Baggage.